internal/clock/now.go

```diff
@@ -0,0 +1,83 @@
+// Package clock provides indirection for accessing current time.
+package clock
+
+import (
+	"encoding/json"
+	"log"
+	"net/http"
+	"os"
+	"sync"
+	"time"
+)
+
+const refreshServerTimeEvery = 3 * time.Second
+
+// Now is overridable function that returns current wall clock time.
```

Comment:
To prevent this from being overridden at runtime in production, would it make sense to put this behind a build flag? such that the default implementation is:

```go
func Now() time.Time {
	return time.Now()
}
```

And an alternate version with a build flag with the one here.

```go
var Now = time.Now
```

Of course, this would require the corresponding build flags for the endurance test.

Reply:
excellent idea

Reply:
done